src/main/java/gift/product/repository/ProductRepository.java

```diff
@@ -0,0 +1,77 @@
+package gift.product.repository;
+
+import gift.common.dto.PagedResult;
+import gift.common.exception.ErrorCode;
+import gift.common.exception.InvalidSortFieldException;
+import gift.product.domain.Product;
+import gift.product.exception.ProductNotFoundException;
+import org.springframework.stereotype.Repository;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicLong;
+
+@Repository
+public class ProductRepository {
+
+    private final ConcurrentHashMap<Long, Product> productMap = new ConcurrentHashMap<>();
+    private final AtomicLong idGenerator = new AtomicLong();
+
+    public PagedResult<Product> findAll(int page, int size, String sortField, boolean asc) {
+        Comparator<Product> comparator = switch (sortField) {
+            case "name" -> Comparator.comparing(Product::getName);
+            case "price" -> Comparator.comparingInt(Product::getPrice);
+            case "id" -> Comparator.comparing(Product::getId);
+            default -> throw new InvalidSortFieldException(ErrorCode.INVALID_INPUT_VALUE);
+        };
+
+        if (!asc) comparator = comparator.reversed();
```

Comment:
```suggestion
        if (!asc) {
            comparator = comparator.reversed();
        }
```

자바 스타일 가이드에 따르면 한줄이더라도 괄호는 생략하지 않는 것이 좋습니다.

Reply:
코드 스타일에 소홀했던 것 같은데 사소한 부분도 캐치해주셔서 감사합니다!!😊